### PR TITLE
feat: merge eth-proxy-contract into eth-input-data 

### PR DESCRIPTION
**Pull request by vrolland**

### CODE REVIEW

File: packages/advanced-logic/specs/payment-network-eth-input-data-0.2.0.md [hunk 1]
```diff
@@ -5,12 +5,18 @@ You may be interested in this document if:
 - you want to create your own implementation of the Request protocol
 - you are curious enough to dive and see what is under the hood of the Request protocol
 
-Prerequisite: Having read the advanced logic specification (see [here](./advanced-logic-specs-0.1.0.md)).
+Prerequisite: Having read the advanced logic specification (see [here](./advanced-logic-specs-0.2.0.md)).
```
Comment (benjlevesque) [inline packages/advanced-logic/specs/payment-network-eth-input-data-0.2.0.md]: why did you bump this file? do you plan on bumping all specs?

File: packages/advanced-logic/specs/payment-network-eth-input-data-0.2.0.md [hunk 2]
```diff
@@ -5,12 +5,18 @@ You may be interested in this document if:
...
 
 ## Description
 
 This extension allows the payments and the refunds to be made in Ether on the Ethereum blockchain.
-A payment reference has to be given in input data when making the transfer to link the payment to the request.
+A payment reference has to be given when making the transfer to link the payment to the request.
+
+There is two ways to add a payment reference to a transfer:
```
Comment (benjlevesque) [inline packages/advanced-logic/specs/payment-network-eth-input-data-0.2.0.md]:
```suggestion
There are two ways to add a payment reference to a transfer:
```

File: packages/advanced-logic/specs/payment-network-eth-input-data-0.2.0.md (hunk 1 above)
Comment (vrolland) [reply packages/advanced-logic/specs/payment-network-eth-input-data-0.2.0.md]: Good one. It was just a mistake.

Comment (rittme) [reply packages/advanced-logic/specs/payment-network-eth-input-data-0.2.0.md]: seems like a search/replace mistake :)

**Pull request merged**